pandas/tools/util.py

```diff
@@ -61,6 +61,42 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : {'signed', 'unsigned'}, str, numpy.dtype, or
+               Python dtype, default None
+        If possible, downcast the resulting numerical data to the
+        specified numerical dtype in 'downcast'. If the size
+        of the data's dtype is smaller or equal to that of 'downcast',
+        then this parameter will be ignored.
+
+        If 'downcast' refers to a more generic dtype (e.g. int),
+        downcast the resulting numerical data to the smallest
+        dtype possible under that more generic dtype specification
+        that is of smaller size than the current numerical
+        data dtype according to the following rules:
```

Comment:
@jreback : I know you have mentioned this multiple times, but see my comment <a href="https://github.com/pydata/pandas/pull/13425#issuecomment-230118332">here<a/>.  Some justification for this behaviour?  Again, I don't see we need to be this harsh.